### ISSUE TITLE
Write tests and set up CI

### ISSUE THREAD

**Issue by symroe** [DemocracyClub/UK-Polling-Stations#140]
Shamefully we don't have any tests for this project – this is really because it was a quite proof of concept that is turning in to a live service.

We should address this techincal debt!

**Comment by symroe** [comment]
CI implemented in 8f808495b1d31cad6f6e12db4f69e98f94d09ae1

**Comment by chris48s** [comment]
I've set up a wiki and added some manual testing resources

* Postcodes which take you on particular paths through the system or trigger various unusual conditions.
* A list of SQL statements I have found very useful for checking imported data.

In the short term, we should add more useful postcodes / testing resources to the wiki as we find them and make use of these when we change stuff.

Stretch goals should be:

* Use this as a basis to create testing fixtures which replicate these conditions and write automated tests to ensure website behaviour is correct and future changes don't break it.
* Use these SQL statements to add a task/report which can be run after importing new data files to flag up possible error conditions to investigate.
